spark_log_parser/loaders/__init__.py

```diff
@@ -123,6 +123,10 @@ class ZipArchiveMemberStreamWrapper(FileChunkStreamWrapper):
     """Just an alias for FileChunkStreamWrapper for use when extracting Zip archive members using stream-unzip"""
 
 
+FileStreamIterator = TypeVar("FileStreamIterator", bound=Iterator[bytes])
+FileExtractionResult = TypeVar("FileExtractionResult", bound=tuple[Path, FileStreamIterator])
```

Comment:
Any idea why Python requires that first string arg to `TypeVar`?

Reply:
I have been wondering about this as well, yeah. Looking at the official docstring, I think it just serves as a way for these vars to be "named", since the `TypeVar` object itself can't access the variable name that's on the left-hand side of the assignment -

```python
    Type variables can be introspected. e.g.:

      T.__name__ == 'T'
      T.__constraints__ == ()
      T.__covariant__ == False
      T.__contravariant__ = False
      A.__constraints__ == (str, bytes)
```

Reply:
Ah. Thanks for sharing!